go.mod

```diff
@@ -1,19 +1,21 @@
 module github.com/kubewarden/safe-labels-policy
 
-go 1.20
+go 1.21
+
+toolchain go1.21.6
```

Comment:
complains of unknown directive toolchain.

Reply:
solved by upgrading to the latest version of our GH actions